openedx/core/djangoapps/content/search/tests/test_api.py

```diff
@@ -975,3 +1061,51 @@ def test_block_in_units(self, mock_meilisearch):
             ],
             any_order=True,
         )
+
+    @override_settings(MEILISEARCH_ENABLED=True)
+    def test_units_in_subsection(self, mock_meilisearch):
+        with freeze_time(self.created_date):
+            library_api.update_container_children(
+                LibraryContainerLocator.from_string(self.subsection_key),
+                [LibraryContainerLocator.from_string(self.unit_key)],
+                None,
+            )
+
+        # TODO verify subsections in units
```

Comment:
What's left to do here? This looks similar to the units case above.

Reply:
It was missing to add to the index the sections and subsections to which it is contained.  Navin has advanced this part in https://github.com/openedx/edx-platform/pull/36762#issuecomment-2916769375. I'll review it and update the tests.